notification/app/notification/services/azure/GCMPushConverter.scala

```diff
@@ -157,7 +157,7 @@ class GCMPushConverter(conf: Configuration) extends AzurePushConverter {
       "awayTeamScore" -> Some(matchStatusAlert.awayTeamScore.toString),
       "awayTeamText" -> Some(matchStatusAlert.awayTeamMessage),
       "currentMinute" -> Some(""),
-      "important" -> Some(matchStatusAlert.importance.toString),
+      "importance" -> Some(matchStatusAlert.importance.toString),
```

Comment:
Are there any side effects to changing important to importance?

Reply:
Yes, it will fix the android side as it's currently not parsing it